ci/build_alpine_container/README.md

```diff
@@ -0,0 +1,17 @@
+This directory contains a Dockerfile that builds an envoy image
```

Comment:
Thanks @mattklein123 @junr03 @rshriram - @junr03 I've added a patch on top of the existing commit to build and push both debug and stripped alpine envoy images.